Hbase docs - cleanup (#521)
stackabletech/hbase-operator: docs/modules/hbase/pages/usage-guide/phoenix.adoc

= Using Apache Phoenix
:phoenix-installation: https://phoenix.apache.org/installation.html
:sqlline-github: https://github.com/julianhyde/sqlline

> The Apache Phoenix project provides the ability to interact with HBase with JBDC using familiar SQL-syntax.

Reviewer: ```suggestion
Apache Phoenix allows you to interact with HBase using a familiar SQL-syntax via a JDBC driver.
```

The Phoenix dependencies are bundled with the Stackable HBase image and do not need to be installed separately (client components will need to ensure that they have the correct client-side libraries available).
Information about client-side installation can be found {phoenix-installation}[here].

Apache Phoenix comes bundled with a few simple scripts to verify a correct server-side installation.
For example, assuming that Phoenix dependencies have been installed to their default location of `/stackable/phoenix/bin`, we can issue the following using the supplied `psql.py` script:

[source,shell]
----
$ /stackable/phoenix/bin/psql.py  && \
   /stackable/phoenix/examples/WEB_STAT.sql && \
   /stackable/phoenix/examples/WEB_STAT.csv  && \
   /stackable/phoenix/examples/WEB_STAT_QUERIES.sql
----

> This script creates a java command that creates, populates and queries a Phoenix table called `WEB_STAT`.

Reviewer: Leave Java out. It's not relevant here.

```suggestion
This script creates, populates and queries a Phoenix table called `WEB_STAT`.
```

Alternatively, one can use the `sqlline.py` script (which wraps the {sqlline-github}[sqlline] utility):

[source,shell]
----
$ /stackable/phoenix/bin/sqlline.py [zookeeper] [sql file]
----

> The script opens an SQL prompt from where one can list, query, create and generally interact with Phoenix tables.

Reviewer: ```suggestion
`sqlline.py` provides an SQL prompt where you can work Phoenix tables interactively.
```

The `SYSTEM`* tables are those required by Phoenix and are created the first time that Phoenix is invoked.

> NOTE: Both `psql.py` and `sqlline.py` generate a java command that calls classes from the Phoenix client library `.jar`.

Reviewer: ```suggestion
NOTE: Both `psql.py` and `sqlline.py` use the Java  client library for Phoenix and spawn a new `java` process in the background.
```
